libtsuba/include/katana/RDGPrefix.h

```diff
@@ -54,6 +54,9 @@ class KATANA_EXPORT RDGPrefix {
       const RDGManifest& manifest, uint32_t partition_id);
 };
 
+/// THIS HEADER WAS DEPRECATED AS OF storage_format_version = 4
+/// DO NOT MODIFY THIS HEADER
+/// This header is used only for backwards compatibility with older storage formats
 /// EntityTypeIDArrayHeader describes the header in the on disk representation
 /// of EntityTypeID arrays, it could be probably be rolled into RDGPrefix but it
 /// has slightly different uses so for now it is separate
```

Comment:
And so it begins...

Reply:
Maybe when we do a full release we can drop support for all older storage_format_versions and start fresh without our old sins to weigh us down. 
That would leave more room for our future sins. :D